package.json

```diff
@@ -30,7 +30,7 @@
     "ccxt": "^1.10.171",
     "express": "^4.16.2",
     "forex.analytics": "mkmarek/forex.analytics#7bc278987700d4204e959af17de61495941d1a14",
-    "gdax": "^0.4.2",
+    "gdax": "coinbase/gdax-node",
```

Comment:
It might be a little more resilient to say `coinbase/gdax-node#semver:^0.5.0`

Reply:
Not a bad suggestion at all